### PR TITLE
ff7tk-0.83-stage

### PR DESCRIPTION
**Pull request by sithlord48**
- Use Alias Style Includes
 - Ask ff7tk for its translations path
 - Bump ff7tk depend to 0.83

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -70,19 +70,13 @@ add_compile_definitions(
...
+find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Widgets OpenGLWidgets Svg OpenGL LinguistTools Core5Compat Xml Quick REQUIRED)
 
-set(REQUIRED_FF7TK_VERSION 0.82.0)
-find_package(ff7tk ${REQUIRED_FF7TK_VERSION} REQUIRED COMPONENTS ff7tk ff7tkQtWidgets ff7tkFormats ff7tkUtils)
+set(REQUIRED_FF7TK_VERSION 0.83.0)
+find_package(ff7tk ${REQUIRED_FF7TK_VERSION} REQUIRED COMPONENTS ff7tk fftkData ff7tkQtWidgets ff7tkFormats ff7tkUtils)
 
 find_package(ZLIB REQUIRED)
 
-if(${QT_VERSION_MAJOR} EQUAL 6)
```
Comment (myst6re) [inline CMakeLists.txt:81]: There no need to remove this if, please keep it

Comment (myst6re) [reply CMakeLists.txt:81]: Or yes remove it, it is not important

Comment (sithlord48) [reply CMakeLists.txt:81]: We can't build with Qt5 anymore

**Comment by myst6re** [comment]
Thanks!

**Pull request merged**